src/werkzeug/wrappers/request.py

```diff
@@ -13,6 +14,7 @@ class Request(
     UserAgentMixin,
     AuthorizationMixin,
     CommonRequestDescriptorsMixin,
+    RequestCORSMixin,
```

Comment:
`CommonRequestDescriptors` should probably be last.

Reply:
Done.